### DISCUSSION TITLE
When will 1.7.2 be released?

### DISCUSSION THREAD

**Discussion by marchpane** [resilience4j/resilience4j#1676] [Q&A] [answered]
Hi folks,

I'm curious to know how this works in regards to a new release.
 
I ask because I see some usefull pull requests being merged after 1.7.1 and I wonder when 1.7.2 will be released so that the world would be able to use new features?

I would really appreciate if anybody could shed some ligth to this topic.

Thank you!

**Comment by RobWin** [comment, marked as answer]
Hi,
we are not planning to release any 1.7.x release anymore.
We are focused on releasing 2.x based on Java 11.

Most of the PR have been only merged into the master (2.x) branch and not backported to 1.7.x branch
